### PR TITLE
Add PyTorch BERT script mode example

### PR DESCRIPTION
**Pull request by jkroll-aws**
*Issue #, if available:* n/a

*Description of changes:* Move existing PyTorch BERT script mode example into this repo.

Original source: https://github.com/aws-samples/amazon-sagemaker-script-mode/blob/master/deploy-pretrained-model/BERT/Deploy_BERT.ipynb

Future notebook link: https://github.com/aws/amazon-sagemaker-examples/tree/main/sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb

*Testing done:* Ran end-to-end in notebook instance.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb [hunk 1]
```diff
@@ -0,0 +1,226 @@
+{
```
Comment (jkroll-aws) [inline sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb:1]: <p>Typo: <code>predicitions</code> -&gt;  <code>predictions</code>	, I'll fix. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Review by jeniyat — APPROVED** [review]

File: sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb (hunk 1 above)
Comment (atqy) [inline sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb:1]: <p>We should switch delete_endpoint and delete_model as that has caused error in the past before. But it is interesting that CI still passed with this ordering - it used to fail</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (stacicho) [reply sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb:1]: <p>test reply</p>

Comment (jkroll-aws) [reply sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb:1]: Switched order to delete the model first, then endpoint.

Comment (jkroll-aws) [reply sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb:1]: Fixed typo.

**Review by atqy — APPROVED** [review]

**Review by jeniyat — APPROVED** [review]

**Pull request merged**